protocol/v2/types/sync_committee.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+)
+
+type BlockRootWithSlot struct {
+	spectypes.SSZBytes // block root
+	Slot               phase0.Slot
+}
```

Comment:
if this is a type solely meant for signing, perhaps this should be in `ekm` package

also, let's please use phase0.Root instead of an interface that could be other things too

Reply:
done